content/v2.markdown

```diff
@@ -271,7 +271,7 @@ List all domains in the account `1010` sorting them by expiration date (asc) and
 
     curl  -H 'Authorization: Bearer <token>' \
           -H 'Accept: application/json' \
-          https://api.dnsimple.com/v2/1010/domains?sort=expires_on:asc,id:desc
```

Comment:
This is a breaking change. Therefore, we'll have to alias `expires_on` in our app to continue to work.

Our API should not be tied to the internal of our system. We used "modification", we should have used "expiration" here. I suggest you change the filter to `expiration`, and in our app we'll both handle `expires_on` and `expiration`.

Reply:
`expires_on` and `expires_at` is both working as a filter, but I agree with you that `expiration` is a better name for the filter